contents/docs/data/events.mdx

```diff
@@ -43,6 +43,12 @@ PostHog is event-based, meaning much of the functionality is triggered through e
 
 - Sending data to webhooks is triggered by ingesting specific events.
 
+## Anonymous vs identified events
+
+PostHog captures two types of events: **anonymous** and **identified**. Identified events enable you to attribute events to specific users, while anonymous events do not contain individually identifiable data.
```

Comment:
You could add PII as a event property to an event, so saying "anonymous events do not contain individually identifiable data" isn't 100% true.

```suggestion
PostHog captures two types of events: **anonymous** and **identified**. Identified events enable you to attribute events to person profiles, while anonymous events do not.
```